[2021-04-23 16:42:52,729] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.delete_cluster_dataproc 2021-04-23T15:00:00+00:00 [queued]>
[2021-04-23 16:42:52,796] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.delete_cluster_dataproc 2021-04-23T15:00:00+00:00 [queued]>
[2021-04-23 16:42:52,799] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-23 16:42:52,802] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-04-23 16:42:52,804] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-23 16:42:52,823] {taskinstance.py:1063} INFO - Executing <Task(PatchedDataprocDeleteClusterOperator): delete_cluster_dataproc> on 2021-04-23T15:00:00+00:00
[2021-04-23 16:42:52,842] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'recent_played_ingestion', 'delete_cluster_dataproc', '2021-04-23T15:00:00+00:00', '--job-id', '15', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/recent_played_ingestion.py', '--cfg-path', '/tmp/tmplyz9g1zw', '--error-file', '/tmp/tmpo7j6ku8p']
[2021-04-23 16:42:52,835] {standard_task_runner.py:52} INFO - Started process 97 to run task
[2021-04-23 16:42:52,846] {standard_task_runner.py:77} INFO - Job 15: Subtask delete_cluster_dataproc
[2021-04-23 16:42:52,933] {logging_mixin.py:104} INFO - Running <TaskInstance: recent_played_ingestion.delete_cluster_dataproc 2021-04-23T15:00:00+00:00 [running]> on host 760ffd3d790b
[2021-04-23 16:42:53,029] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=recent_played_ingestion
AIRFLOW_CTX_TASK_ID=delete_cluster_dataproc
AIRFLOW_CTX_EXECUTION_DATE=2021-04-23T15:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-23T15:00:00+00:00
[2021-04-23 16:42:53,050] {dataproc.py:840} INFO - Deleting cluster: ephemeral-recent-played-spark-cluster-20210423
[2021-04-23 16:43:34,568] {dataproc.py:852} INFO - Cluster deleted.
[2021-04-23 16:43:34,609] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=recent_played_ingestion, task_id=delete_cluster_dataproc, execution_date=20210423T150000, start_date=20210423T164252, end_date=20210423T164334
[2021-04-23 16:43:34,698] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-04-23 16:43:34,744] {local_task_job.py:146} INFO - Task exited with return code 0
